src/software/ai/navigator/path_planner/theta_star_path_planner_test.cpp

```diff
@@ -287,8 +287,8 @@ TEST(TestThetaStarPathPlanner, performance)
 
     std::chrono::duration<double> duration = end_time - start_time;
 
-    std::chrono::duration<double> avg =
-        duration / ((double)num_iterations * obstacle_sets.size() - 1);
+    std::chrono::duration<double> avg;
+    avg = duration / ((double)num_iterations * obstacle_sets.size() - 1);
```

Comment:
just curious, what warning does this fix?

Reply:
Looks like it's unused. I'd prefer to just delete it in that case.

Reply:
I saw something similar with Hardik, I think it's to do with implicit `const`-ness of the variable? If it is, does adding the `const` modifier fix it?

Reply:
According to the comments in this specific test `PassingEvaluationTest`, 
>This test does not assert anything. Rather, It can be used to gauge how
    fast ratePass is running, and can be profiled in order to find areas
    of improvement for ratePass

Its just that `avg` is unused when the speed results are not printed (they are commented out). We could comment this out as well, but that would cause `duration` and `start_time` to be unused earlier in the test, causing the same warning. 

We could comment out all the lines that actually measure performance, but I thought that having multiple commented out lines in one test is confusing. 

Alternatively, we could remove all the lines that relate to measuring performance, but then this test serves no purpose.

Reply:
Maybe we can disable the test and leave it all uncommented? Then if we want to profile we can enable it.

https://stackoverflow.com/questions/7208070/googletest-how-to-skip-a-test

Also @garethellis0 thoughts since this was your test?

Reply:
Honestly I'm not a _huge_ fan of this test, it was written before competition out of necessity. I think the `GTEST_SKIP` macro is appropriate here.

Reply:
I agree

Reply:
We should document tests that we are disabling in a central doc

Reply:
Hmm, don't think that's maintainable. I don't see any issue with having a policy of never skipping tests that are actually doing validation work.

Reply:
Yea IMO 99.99999999% of tests should be run, and if one needs to be disabled there should be a ticket to fix whatever needs to be fixed to re-enable it. Agree a doc isn't maintainable.